api/utils/utils.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
+
+type server struct {
+	Address string `json:"address"`
+}
+
+type config struct {
+	Server server `json:"server"`
+}
+
+var Config config
+
+func ReadConfig(path string) (config, error) {
+
+	jsonf, err := os.Open(path)
+
+	if err != nil {
+		return *new(config), err
+	}
+
+	data, err := ioutil.ReadAll(jsonf)
+
+	if err != nil {
+		return *new(config), err
+	}
+
+	json.Unmarshal(data, &Config)
```

Comment:
The unmarshall function already returns an error, you should probably check that also